ui/raidboss/data/04-sb/dungeon/bardams_mettle.txt

```diff
@@ -0,0 +1,169 @@
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 00:0839:.*is no longer sealed/ window 5000 jump 0
+
+#~~~~~~~~#
+# Garula #
+#~~~~~~~~#
+
+# -ic "Steppe Coeurl" "Steppe Sheep" "Steppe Yamaa"
+
+0 "Start" sync / 00:0839:Bardam's Hunt will be sealed off/
+11.7 "Heave" sync /:Garula:1EF7:/ window 11.7,5
+20.9 "Crumbling Crust (cast)" sync /:Garula:1EF8:/
+22.4 "Crumbling Crust (circles)" sync /:Garula:1F13:/
+34.1 "Rush" sync /:Garula:1EF9:/
+36.2 "War Cry" sync /:Garula:1EFA:/
+38.9 "Earthquake" sync /:Garula:1EFB:/ window 15,15
+49.5 "Heave" sync /:Garula:1EF7:/
+58.7 "Crumbling Crust (cast)" sync /:Garula:1EF8:/
+60.2 "Crumbling Crust (circles)" sync /:Garula:1F13:/
+
+66.6 "Heave" sync /:Garula:1EF7:/ jump 11.7
+75.8 "Crumbling Crust (cast)"
+77.3 "Crumbling Crust (circles)"
+89.0 "Rush"
+91.1 "War Cry"
+93.8 "Earthquake"
+
+#~~~~~~~~#
+# BARDAM #
+#~~~~~~~~#
+
+# -ii 1EFE 2578 2579 257A 257E 257B
+
+1000.0 "Start" sync / 00:0839:.*Rebirth of Bardam the Brave will be sealed off/ window 1000,5
+1007.8 "Travail" sync /:Bardam:1EFF:/ window 7.8,5
+1014.1 "Magnetism" sync /:Hunter Of Bardam:1F08:/
+1020.3 "Tremblor" sync /:Bardam:257C:/
+1022.8 "Empty Gaze" sync /:Hunter Of Bardam:1F04:/
+1025.0 "Travail" sync /:Bardam:1EFF:/
```

Comment:
Should Travail / Reconstruct (others) be hidden with a `hideall`?

Reply:
That's reasonable.